integration/cri-o/crio_skip_tests.sh

```diff
@@ -8,36 +8,206 @@
+['test "test infra ctr dropped"']='drop_infra.bats case is not working: see $url'
+['test "test infra ctr not dropped"']='drop_infra.bats case is not working: see $url'
+)
+declare -A image_skipCRIOTests=(
+['test "image pull and list by manifest list digest"']='image.bats case is not working: see $url'
+['test "image pull and list by manifest list tag"']='image.bats case is not working: see $url'
+['test "image pull and list by manifest list and individual digest"']='image.bats case is not working: see $url'
+['test "image pull and list by individual and manifest list digest"']='image.bats case is not working: see $url'
+['test "image pull with signature"']='skip registry has some issues'
+['test "container status when created by image list canonical reference"']='drop_infra.bats case is not working: see $url'
+)
+declare -A infra_ctr_cpuset_skipCRIOTests=(
+['test "test infra ctr cpuset"']='infra_ctr_cpuset.bats case is not working: see $url'
+)
+declare -A inspect_skipCRIOTests=(
+['test "info inspect"']='inspect.bats case is not working: see $url'
+['test "ctr inspect"']='inspect.bats case is not working: see $url'
+['test "pod inspect when dropping infra"']='inspect.bats case is not working: see $url'
+['test "ctr inspect not found"']='inspect.bats case is not working: see $url'
+)
+declare -A metrics_skipCRIOTests=(
+['test "metrics with default port"']='metrics.bats case is not working: see $url'
+['test "metrics with random port"']='metrics.bats case is not working: see $url'
+['test "metrics with operations quantile"']='metrics.bats case is not working: see $url'
+['test "secure metrics with random port"']='metrics.bats case is not working: see $url'
+['test "secure metrics with random port and missing cert\/key"']='metrics.bats case is not working: see $url'
+['test "metrics container oom"']='metrics.bats case is not working: see $url'
+)
+declare -A namespaces_skipCRIOTests=(
+['test "pid namespace mode target test"']='namespaces.bats case is not working: see $url'
+)
+declare -A network_skipCRIOTests=(
+['test "ensure correct hostname for hostnetwork:true"']='network.bats case is not working: see $url'
+['test "Connect to pod hostport from the host"']='network.bats case is not working: see $url'
+['test "Clean up network if pod sandbox fails"']='network.bats case is not working: see $url'
+['test "Clean up network if pod sandbox gets killed"']='network.bats case is not working: see $url'
+)
+declare -A network_ping_skipCRIOTests=(
+['test "Ping pod from the host \/ another pod"']='network_ping.bats case is not working: see $url'
+)
+declare -A pod_skipCRIOTests=(
+['test "restart crio and still get pod status"']='pod.bats case is not working: see $url'
+['test "systemd cgroup_parent correctly set"']='pod.bats case is not working: see $url'
+['test "kubernetes pod terminationGracePeriod passthru"']='pod.bats case is not working: see $url'
+['test "crio restore"']='pod.bats case is not working: see $url'
```

Comment:
"crio restore" is part of "restore.bats", not "pod.bats".
You need to move it to the appropriate list, otherwise it won't be skipped, and it will cause a failure.